src/gcp/cloudscheduler.ts

```diff
@@ -212,3 +213,36 @@ export function jobFromSpec(schedule: backend.ScheduleSpec, appEngineLocation: s
   };
   return job;
 }
+
+/** Converts an Endpoint to a CloudScheduler v1 job */
+export function jobFromEndpoint(
+  endpoint: backend.Endpoint & backend.ScheduleTriggered,
+  appEngineLocation: string
+): Job {
+  const job: Partial<Job> = {};
+  if (endpoint.platform === "gcfv1") {
+    const id = backend.scheduleIdForFunction(endpoint);
+    const region = appEngineLocation;
```

Comment:
q: Is it possible and okay for `endpoint.region` to be different than `appEngineLocation`?